appveyor.bat

```diff
@@ -57,7 +57,7 @@ set RUBY_DIR=!RUBY%BIT%_DIR!
 set TCL_VER_LONG=8.6
 set TCL_VER=%TCL_VER_LONG:.=%
 set TCL32_URL=http://downloads.activestate.com/ActiveTcl/releases/8.6.6.8607/ActiveTcl-8.6.6.8607-MSWin32-x86-403667.exe
-set TCL64_URL=http://downloads.activestate.com/ActiveTcl/releases/8.6.6.8606/ActiveTcl-8.6.6.8606-MSWin32-x64-401995.exe
+set TCL64_URL=https://camel-builds.s3.amazonaws.com/ActiveTcl/MSWin32-x64/20200901T013213Z/ActiveTcl-8.6.10.0000-MSWin32-x64-893162d6.exe?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJAFTYUXEZJ3HWLEQ%2F20200909%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20200909T132036Z&X-Amz-Expires=21600&X-Amz-SignedHeaders=host&X-Amz-Signature=53a2277f96134f76d0e2d97212ec7b1fc73a4b4ab83e977b9fe7dc927d08a3de
```

Comment:
Is it okay to write the URL directly here?
Don't we need to use secrets?

Reply:
I wondered about it, but did not find anything about it. So I just copy-pasted the url